### PR TITLE
Add marathon service labels to Consul KV

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
Not ready for review yet.

**Comment by smn** [comment]
@miltontony & @Rizziepit ready for review.

File: consular/tests/test_main.py [hunk 1]
```diff
@@ -230,8 +257,25 @@ def test_sync_app_task(self):
...
+    def test_sync_app_labels(self):
+        app = {
+            'id': '/my-app',
+            'labels': {'foo': 'bar'}
+        }
+        d = self.consular.sync_app_labels(app)
+        consul_request = yield self.consul_requests.get()
+        self.assertEqual(consul_request['method'], 'PUT')
+        self.assertEqual(consul_request['path'], '/v1/kv/consular/my-app/foo')
+        self.assertEqual(consul_request['data'], 'bar')
```
Comment (miltontony) [inline consular/tests/test_main.py:269]: shouldn't this be in a key call `foo` instead of `data`?

Comment (smn) [inline consular/tests/test_main.py:269]: Yeah `foo` is the key, it's in the `'path'` I'm checking the line above.
The `consul_request` dict here is the mocked call to Consul, `data` here is the payload being sent in the HTTP request, which in this case is `bar`.

Comment (miltontony) [inline consular/tests/test_main.py:269]: gotcha :ok_hand:

**Comment by miltontony** [comment]
:+1:

**Pull request merged**